src/main/java/com/smartbear/ready/plugin/jira/actions/CreateNewBugAction.java

```diff
@@ -341,10 +352,17 @@ private void addExtraFields(XForm baseDialog, JiraProvider bugTrackerProvider, S
             CimFieldInfo fieldInfo = field.getValue();
             XFormField newField;
             if (fieldInfo.getAllowedValues() != null) {
-                Object[] values = IterableObjectsToNameArray(bugTrackerProvider, fieldInfo.getAllowedValues(), !fieldInfo.isRequired());
+                boolean isMultiValueCustomField = isSpecifiedTypeCustomField(fieldInfo, MULTICHECKBOXES_CUSTOM_FIELD_TYPE) ||
+                        isSpecifiedTypeCustomField(fieldInfo, MULTISELECT_CUSTOM_FIELD_TYPE);
+                boolean addEmptyValue = isMultiValueCustomField ? false : !fieldInfo.isRequired();
```

Comment:
Will it work fine if a custom field is required?

Reply:
Yes. User must select at least one value